instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapper.java

```diff
@@ -159,23 +157,19 @@ void captureMdcAttributes(AttributesBuilder attributes, Map<String, String> mdcP
 
     if (captureAllMdcAttributes) {
       for (Map.Entry<String, String> entry : mdcProperties.entrySet()) {
-        attributes.put(getMdcAttributeKey(entry.getKey()), entry.getValue());
+        attributes.put(AttributeKey.stringKey(entry.getKey()), entry.getValue());
       }
       return;
     }
 
     for (String key : captureMdcAttributes) {
       String value = mdcProperties.get(key);
       if (value != null) {
-        attributes.put(getMdcAttributeKey(key), value);
+        attributes.put(AttributeKey.stringKey(key), value);
       }
     }
   }
 
-  public static AttributeKey<String> getMdcAttributeKey(String key) {
```

Comment:
why did you decide to remove the attribute cache?

Reply:
I assumed it was required only because of prefix addition, but now since we don't do it anymore it shouldn't be required, since the rest places there attributes are created aren't using the cache.

Reply:
It was used to reduce creation of `AttributeKey` instances

Reply:
Alright, I'll rollback the change

Reply:
rolled back